deployments/demos/local-containers/README.md

```diff
@@ -0,0 +1,209 @@
+# OpenADKit Remote Control Architecture: Implementation Report and User Manual
+
+## 1. Introduction
+
+This document provides a comprehensive implementation guide and technical overview for the distributed architecture of the OpenADKit project. The core objective is to separate compute-intensive and lightweight components of an autonomous driving system, enabling deployment across different hardware. For example, the core Autoware software stack runs on a high-performance cloud server, while simulators and visualization tools operate on a local developer's laptop.
```

Comment:
The use case is running Autoware on the local side (e.g., vehicle). Users can remotely visualize and manage Autoware from the cloud-based management system.